packages/react-dom/src/__tests__/ReactDOM-test.js

```diff
@@ -308,14 +308,15 @@ describe('ReactDOM', () => {
       ReactDOM.render(
         <div>
           <h1>Auto-focus Test</h1>
-          <input autoFocus={true} />
+          <input defaultValue="foo" autoFocus={true} />
           <p>The above input should be focused after mount.</p>
         </div>,
         container,
       );
 
       expect(inputFocusedAfterMount).toBe(true);
       expect(focusedElement.tagName).toBe('INPUT');
+      expect(focusedElement.selectionStart).toBe(0);
```

Comment:
Are you sure this test is useful?  It seems this assertion passes even without a fix to the actual problem.